include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -209,35 +222,38 @@ inline void StatsdClient::count(const std::string& key,
                                 const int delta,
                                 float frequency,
                                 const std::vector<std::string>& tags) const noexcept {
-    return send(key, delta, detail::METRIC_TYPE_COUNT, frequency, tags);
+    send(key, delta, detail::METRIC_TYPE_COUNT, frequency, tags);
```

Comment:
ha good catch!

Reply:
Indeed :)
The same can be done to `decrement` and `increment`.